CHANGELOG.md

```diff
@@ -15,10 +15,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Several minor issues in documentation
+- Private `Campaign` attributes are now private and no longer exposed via constructor
```

Comment:
That reads really strange since I'd understand this as "they were private before, but now they are private!" Maybe something like "Fix visibility and exposure via constructor of `Campaign` attributes which should be private"?

Reply:
Thanks for the suggestion, will change. Was struggling to find a short sentence that explains it :D